### PR TITLE
[ENH] add OR pediatric bundle definition, make it easier to add future pediatric bundles

### PR DESCRIPTION
**Pull request by 36000**

### CODE REVIEW

File: AFQ/api/bundle_dict.py [hunk 1]
```diff
@@ -740,3 +740,57 @@ def load_templates(self):
...
+                "cross_midline": False
+            }
+        elif bundle_name == "OR_R":
+            self._dict["OR_R"] = {
+                "include": [self.templates["OR_right_roi3"]],
+                "start": self.templates["OR_rightThal"],
+                "end": self.templates["OR_rightV1"],
+                "cross_midline": False
+            }
+        elif bundle_name == "pARC_L":
```
Comment (arokem) [inline AFQ/api/bundle_dict.py:759]: Do we want to add at least pARC and VOF to the default set that always gets run?

**Comment by 36000** [comment]
image
@arokem  this is ready for review

File: AFQ/api/bundle_dict.py (hunk 1 above)
Comment (36000) [reply AFQ/api/bundle_dict.py:759]: This might be something to ask Mareike, but I think this is fine for now

**Review by arokem — APPROVED** [review]
LGTM! The CI failures seem unrelated. At some point we can investigate memory management a bit more, but I'll go ahead and merge this for now.

**Comment by arokem** [comment]
@grotheer: should we add at least pARC and VOF to the default set of bundles for babyAFQ?

**Pull request merged**